src/worldcereal/job.py

```diff
@@ -339,5 +344,11 @@ def collect_inputs(
     inputs.execute_batch(
         outputfile=output_path,
         out_format="NetCDF",
-        job_options={"driver-memory": "4g", "executor-memoryOverhead": "4g"},
+        job_options={
```

Comment:
Isn't the presto compile feature only available for more modern python versions than 3.8? If yes how do we specify which UDF environments to use?

Reply:
It's not the python version itself that is the issue on openEO, it's the fact that their 3.8 image does not contain a compiler. I'm not sure if we need to take care of that here? Seems more an openEO thing. In the backlog is a task to create dependencies for 3.11 so we can test it on openEO.